### PR TITLE
Add support for macOS 10.14

### PR DESCRIPTION
**Pull request by ospfranco**
Hi! thanks a lot for the adapter, I'm working on a cross platform application using the newly released react-native-macos and rxdb, it seems the old adapter is now abandoned and full of bugs, so I needed support for macos, just adding it to the pod spec, seems to work!

### CODE REVIEW

**Comment by craftzdog** [comment]
Thanks. Can you preserve the code style?

**Comment by craftzdog** [comment]
Merged it anyway. Will format it later.

File: src/SQLiteDatabase.js [hunk 1]
```diff
@@ -62,7 +66,7 @@ function escapeBlob(data) {
 
 function unescapeForIOSAndAndroid(rows) {
```
Comment (craftzdog) [inline src/SQLiteDatabase.js:67]: Oh, doesn't it have to unescape as well for macOS?

Comment (ospfranco) [reply src/SQLiteDatabase.js:67]: ah.. I'm not 100% sure, right now I haven't found any issues, but it probably does, do you know what do I have to do to run this function?

Comment (ospfranco) [reply src/SQLiteDatabase.js:67]: I created #84, I will continue testing on my application but seems to be fine for now

**Comment by mouhsinelonly** [comment]
i have tested it on an app i am working on . eveyrthing is good until now.

**Comment by craftzdog** [comment]
You can test it with [test/app.js](https://github.com/craftzdog/react-native-sqlite-2/blob/master/test/app.js).
The test tries to insert data with '\0' here: https://github.com/craftzdog/react-native-sqlite-2/blob/master/test/app.js#L185
It triggers escaping/unescaping data.
And you should see the result like this:

Screen Shot 2020-07-22 at 11 47 57

**Pull request merged**